Content.Shared/LandMines/LandMineComponent.cs

```diff
@@ -10,4 +10,11 @@ public sealed partial class LandMineComponent : Component
     /// </summary>
     [DataField, ViewVariables(VVAccess.ReadWrite)]
     public SoundSpecifier? Sound;
+
+    /// <summary>
+    /// Is the land mine armed and dangerous?
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
    [DataField]
```
`DataField` includes VV readwrite nowadays, so this is no longer needed.
The datafield and component should be predicted and networked. For that add the `NetworkedComponent`, `AutoGenerateComponentState` and `AutoNetworkedField` attributes.

Reply:
Fixed